scripts/src/pushowners/owners.py

```diff
@@ -0,0 +1,109 @@
+import argparse
+import re
+from github import Github
+import yaml
+import sys
+import analytics
+sys.path.append('../')
+
+def getFileContent(files):
+    users_included="No"
+    provider_delivery="No"
+    vendor_name=""
+    chart_name=""
+    vendor_type=""
+    for changed_file in files:
+        # Load the YAML file
+        with open(changed_file) as file:
+            documents = yaml.full_load(file)
+            for key, value in documents.items():
+                if key=='providerDelivery' and value=='True':
+                    provider_delivery="Yes"
+                elif key=='users' and len(key)!=0:
+                    users_included="Yes"
+                elif key=='vendor':
+                    vendor_name=value['name']
+                elif key=='chart':
+                    chart_name=value['name']
```

Comment:
I would like this code to make use of,  and be in,  scripts/owners/owners_file.py which is meant to be a one stop shop for all things connected to OWNERS files. It loads the files etc, so add some methods there which return the fields you want. This way if the OWNERS file format ever changes (e.g: the key names change) the only thing we need to change is owners_file.py.

Reply:
Makes sense